sb.web.app/src/components/CreateArticle/Combobox.vue

```diff
@@ -75,8 +75,7 @@ export default {
           day: '2-digit'
         }
       }
-      const now = new Date().toLocaleString('sv-SE', options).replaceAll('-', '/')
-      return now + ' - ' + new Date(date).toLocaleString('sv-SE', options).replaceAll('-', '/')
+      return new Date(date.split('T')[0]).toLocaleDateString('en-US', { month: 'numeric', day: 'numeric', year: 'numeric' })
```

Comment:
hmmm .. switching to en-US date formatted dates - does that mean month/day/year ?

I guess its the right call here but ideally date formatting would be in the organisation configuration.. im gonna add an issue for the way back log